### PR TITLE
python3Packages.radon: init at 5.1.0

### PR DESCRIPTION
**Pull request by jpetrucciani**
###### Description of changes

I would like to be able to use the `radon` ([link here](https://github.com/rubik/radon)) code metrics tool inside of nixpkgs!

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [X] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/mando/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    # this rev contains a fix for newer versions of python (3.8+), but is still tagged 0.6.4
+    rev = "460c2ed296f04530a84d39c1e547b143f11b7580";
+    sha256 = "sha256-Ge73kNrE5PeiFKy1lxOC3OS/0b5HWfGz95ZuLcMJCkI=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+  disabledTests = [
```
Comment (onny) [inline pkgs/development/python-modules/mando/default.nix:26]: Please add a note why these tests have to be disabled

**Review by onny — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/mando/default.nix (hunk 1 above)
Comment (jpetrucciani) [reply pkgs/development/python-modules/mando/default.nix:28]: good call! Done

**Comment by jpetrucciani** [comment]
@onny not sure why it didn't update, but I've added something per your request

**Comment by jpetrucciani** [comment]
This has switched `checkInputs` to `nativeCheckInputs` to satisfy the recent tree-wide change (#206742)! It also adds changelog links, and specifies `format = "setuptools";` explicitly for both packages.

**Pull request closed without merging**